src/httpClient.ts

```diff
@@ -65,6 +65,7 @@ export default class httpClient {
     }
     
     async getAirtableRecords(filters: Filters,
+        only_explore_columns=false,
```

Comment:
underscore king over here, I do want to start enforcing camel casing on the frontend